MovieQuiz/Services/QuestionFactory.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+import UIKit
+
+
+class QuestionFactory: QuestionFactoryProtocol {
+    init(moviesLoader: MoviesLoading, delegate: QuestionFactoryDelegate) {
+        self.delegate = delegate
+        self.moviesLoader = moviesLoader
+    }
+    private let delegate: QuestionFactoryDelegate
+    private let moviesLoader: MoviesLoading
+    private var movies: [MostPopularMovie] = []
```

Comment:
Предпочтительно сначала перечислять все свойства класса или структур, потом перечислять функции, начиная с инита. При этом отделять пустой строкой каждую новую функцию


```suggestion
    private let delegate: QuestionFactoryDelegate
    private let moviesLoader: MoviesLoading
    private var movies: [MostPopularMovie] = []
    
    init(moviesLoader: MoviesLoading, delegate: QuestionFactoryDelegate) {
        self.delegate = delegate
        self.moviesLoader = moviesLoader
    }

```

Reply:
Подробнее можно почитать здесь:

- [Подробное описание codestyle](https://google.github.io/swift/) от Google
- [От Ray Wanderlich](https://github.com/raywenderlich/swift-style-guide)
- [На русском от команды tutu](https://github.com/tutu-ru/swift-style-guide)